server/services/gameLogic.js

```diff
@@ -18,50 +18,34 @@ let roundNum = 0;
 let currentPlayer;
 let currentWord = '';
 
-/* {
-    username,
-    cards,
-    score
-} */
+/* { username, cards, score } */
 let players = [];
 
-/* {
-    username,
-    cardId
-} */
-let cards = [];
+/* { username, cardId } */
+let playedCards = [];
 
-/* {
-    username,
-    cardId
-} */
+/* { username, cardId } */
 let votes = [];
 
-/* Get the current word of the game */
-exports.getWord = () => currentWord;
-
-/* Get the current status of the game */
-exports.getStatus = () => status;
+/* Get the list of cards for a specific player */
+const getCardsByUsername = (username) => players.find(player => player.username === username).cards;
 
-/* Get the current round number */
-exports.getRoundNumber = () => roundNum;
+/* Returns true if this is the current player */
+exports.isCurrentPlayer = (username) => currentPlayer.username === username;
 
-/* Get the current status of the game */
-exports.getCurrentPlayer = () => currentPlayer;
-
-/* Get the list of cards for a specific player */
-exports.getCardsByUsername = (username) => players.find(player => player.username === username).cards.filter(card => card.played === false);
+/* Get the list of unplayed cards for a specific player */
+exports.getUnplayedCardsByUsername = (username) => players.find(player => player.username === username).cards.filter(card => card.played === false);
```

Comment:
So are we sticking with the 'played' field? Or do you wanna add another array in the server which stores all the cards that have been played so far in all the rounds?

Reply:
Hmm, I'm happy to stick with the played field for now until we need something else, if that's good with you?

Reply:
They both seem like equivalent approaches to me, so yeah happy to stick with it too since you're not sending that field back :)